### ISSUE TITLE
What is mq, vq coef for HunyuanVideo T2V to reproduce the results in the paper?

### ISSUE THREAD

**Issue by qchenclaire** [XueZeyue/DanceGRPO#92]
Hello, thanks for sharing the code. I see in finetune_hunyuan_grpo.sh it's only training vq. But in the paper it's multi-reward training. Can you share mq, vq coef?

**Comment by XueZeyue** [comment]
Hi, thanks for your interest,
The results in the paper were obtained by training with VQ and MQ individually. However, joint training would also yield similar results.

The reason I only open-sourced VQ is that MQ is not an effective reward model for visualization purposes. VQ performs well in both visualization and reward improvement, whereas MQ excels at reward improvement but performs poorly in visualization.

An important observation is that VQ and MQ are not decoupled. This means that improvements in VQ will lead to corresponding improvements in MQ.

If you want to implement a joint training paradigm, you simply need to sum up the advantage. (I don’t remember the FPS for MQ exactly, but it might be 12 or 8.)
